### ISSUE TITLE
ADT - Azure Digital Twin Icon Outdated

### ISSUE THREAD

**Issue by tux2323** [plantuml-stdlib/Azure-PlantUML#24]
The icon for the **Azure Digital Twin** (ADT) is not up to date!

**IS:**
![Azure Digital Twin Icon PlantUML Azure](https://raw.githubusercontent.com/plantuml-stdlib/Azure-PlantUML/master/dist/InternetOfThings/AzureDigitalTwins.png)

**SHOULD:**
![Azure Digital Twin Icon ](https://docs.microsoft.com/en-us/answers/topics/25491/icon.html?t=168451)

In my code for the diagram I used this code snipped to include the icon

```plantuml
!define AzurePuml https://raw.githubusercontent.com/plantuml-stdlib/Azure-PlantUML/master/dist/
!includeurl AzurePuml/AzureCommon.puml
!includeurl AzurePuml/InternetOfThings/AzureDigitalTwins.puml
```

Would be super cool if you update the Azure Icons!

**Comment by Potherca** [comment]
@tux2323 Thank you for bringing this to our attention! I've taken the liberty of moving this issue from [C4-PlantUML](https://github.com/plantuml-stdlib/C4-PlantUML) to [Azure-PlantUML](https://github.com/plantuml-stdlib/Azure-PlantUML/), as this is where this will need to be addressed.

Activity on this repo is low so, to manage expectation, do not expect it to be resolved _soon_. That said, activity is expected to pick up soon-ish, so this should be resolved _eventuallly_.

**Comment by Potherca** [comment]
Closed in favor of #29
